js/views/settingsVw.js

```diff
@@ -91,6 +114,15 @@ module.exports = Backbone.View.extend({
     this.moderatorCount = 0;
 
     this.fetchModel();
+    
+    $(window).off('keydown.view').bind('keydown.view', function(e) {
+      var char = String.fromCharCode(e.which).toLowerCase(),
+          ctrl = (e.ctrlKey || e.metaKey) && !e.altKey;
+
+      if (ctrl && char == config.keyShortcuts.save) {
+        window.obEventBus.trigger('saveCurrentForm', self._state);
```

Comment:
Why are we triggering a global event within a view that is then handled by the same view? Global events (which ideally should be used very judiciously) are if we need this view to communicate to other parts of code outside of the view. Why not instead of firing the event, just put the code in the event handler directly here?